lib/core/src/lib/notifications/components/notification-history.component.html

```diff
@@ -1,79 +1,79 @@
-<div (keyup)="onKeyPress($event)" tabindex="-1" role="button" class="adf-notification-history-container">
-    <button mat-button
-            [matMenuTriggerFor]="menu"
-            aria-hidden="false"
-            [attr.aria-label]="'NOTIFICATIONS.OPEN_HISTORY' | translate"
-            title="{{ 'NOTIFICATIONS.OPEN_HISTORY' | translate }}"
-            class="adf-notification-history-menu_button"
-            id="adf-notification-history-open-button"
-            (menuOpened)="onMenuOpened()">
-        <mat-icon matBadge="&#8288;"
-                  [matBadgeHidden]="!notifications.length"
-                  class="adf-notification-history-menu_button-icon"
-                  matBadgeColor="accent"
-                  matBadgeSize="small">notifications
-        </mat-icon>
-    </button>
-    <mat-menu #menu="matMenu"
-              [xPosition]="menuPositionX"
-              [yPosition]="menuPositionY"
-              id="adf-notification-history-menu"
-              class="adf-notification-history-menu adf-notification-history-menu-panel">
-        <div class="adf-notification-history-list"
-             role="button"
-             tabindex="0"
-             (keyup.enter)="$event.stopPropagation()"
-             (click)="$event.stopPropagation()">
-            <div mat-subheader role="menuitem">
-                <span class="adf-notification-history-menu-title">{{ 'NOTIFICATIONS.TITLE' | translate }}</span>
-                <button *ngIf="notifications.length"
-                        id="adf-notification-history-mark-as-read"
-                        class="adf-notification-history-mark-as-read"
-                        mat-icon-button
-                        title="{{ 'NOTIFICATIONS.MARK_AS_READ' | translate }}"
-                        (click)="markAsRead()">
-                    <mat-icon>done_all</mat-icon>
-                </button>
-            </div>
+<button mat-button
+        [matMenuTriggerFor]="menu"
+        aria-hidden="false"
+        [attr.aria-label]="'NOTIFICATIONS.OPEN_HISTORY' | translate"
+        title="{{ 'NOTIFICATIONS.OPEN_HISTORY' | translate }}"
+        class="adf-notification-history-menu_button"
+        id="adf-notification-history-open-button"
+        (menuOpened)="onMenuOpened()">
+    <mat-icon matBadge="&#8288;"
+              [matBadgeHidden]="!notifications.length"
+              class="adf-notification-history-menu_button-icon"
+              matBadgeColor="accent"
+              matBadgeSize="small">notifications
+    </mat-icon>
+</button>
+
+<mat-menu #menu="matMenu"
+          [xPosition]="menuPositionX"
+          [yPosition]="menuPositionY"
+          id="adf-notification-history-menu"
+          class="adf-notification-history-menu adf-notification-history-menu-panel">
+    <div class="adf-notification-history-list"
```

Comment:
here we are getting some eslint issue: 
<img width="953" alt="image" src="https://github.com/user-attachments/assets/d41959b8-0415-4a46-8856-19209a610f00" />